documentation/source/conf.py

```diff
@@ -193,7 +185,7 @@
 }
 
 
-def typehints_formatter(annotation: Any, config: sphinx.config.Config) -> str | None:
+def typehints_formatter(annotation: Any, config: sphinx.config.Config) -> Optional[str]:
```

Comment:
Not sure how I feel about this being optional. Would probably prefer `Union` just so all uses of optional are those with a default value of `None`